docs/pages/admin-guides/access-controls/idps/saml-microsoft-entra-external-id.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Access Azure Portal and CLI 
+description: Access Azure Portal and CLI by authenticating with Teleport SAML IdP
+---
+
+This guide explains how to integrate Teleport SAML IdP with [Microsoft Entra External ID](https://learn.microsoft.com/en-us/entra/external-id/)
+so that users can access Azure Portal and CLI by authenticating with Teleport.
```

Comment:
```suggestion
so that users can access Azure Portal and the Azure CLI by authenticating with Teleport.
```

As far as I understand it, "Azure Portal" is a product name (with no article), while "the Azure CLI" is a common noun and takes an article.